### PR TITLE
Bugfix: remove referee type from support interface if not set

### PR DESCRIPTION
**Pull request by frankieroberto**
The type of reference wasn't asked early on, so some references don't have this field set. This was throwing an error in the support interface.

This is fixed by removing the whole row if there is no reference type to display.

Fixes bug: https://sentry.io/organizations/dfe-teacher-services/issues/3717069711/?project=1765973

### CODE REVIEW

**Review by carlosmartinez — APPROVED** [review]

File: spec/components/support_interface/reference_with_feedback_component_spec.rb [hunk 1]
```diff
@@ -83,6 +83,17 @@
     end
   end
 
+  context 'when there is no refere_type saved (as it’s an old reference)' do
```
Comment (duncanjbrown) [inline spec/components/support_interface/reference_with_feedback_component_spec.rb:86]:
```suggestion
  context 'when there is no referee_type saved (as it’s an old reference)' do
```

**Comment by duncanjbrown** [comment]
I'm happy if @carlosmartinez is happy!

File: spec/components/support_interface/reference_with_feedback_component_spec.rb (hunk 1 above)
Comment (frankieroberto) [reply spec/components/support_interface/reference_with_feedback_component_spec.rb:86]: 🤦

**Comment by frankieroberto** [comment]
@carlosmartinez one fail on this still: "Build and Deploy / Collate test coverage reports (pull_request)" - can that be ignored for now, or is there a fix?

**Comment by tomas-stefano** [comment]
I think is relate to https://github.com/dawidd6/action-download-artifact/issues/207 
I will downgrade on our build to fix the issue.

**Pull request merged**